src/lib/ui-element.coffee

```diff
@@ -0,0 +1,18 @@
+class UIElement
```

Comment:
What kind of logic might we add in the future? If we can;t think of anything then we probably don't need the base class yet.

Either way I would throw an error if it managed to full through?